engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41120to41200.java

```diff
@@ -52,7 +57,15 @@ public boolean supportsRollingUpgrade() {
 
     @Override
     public void performDataMigration(Connection conn) {
+        updateManagementServerHostUuid(conn);
+    }
 
+    private void updateManagementServerHostUuid(Connection conn) {
+        try (final PreparedStatement updateStatement = conn.prepareStatement("UPDATE cloud.mshost SET uuid=UUID()")) {
```

Comment:
shouldn't you use a `where` clause here? I mean, you only want to update the hosts that have the `uuid == null`, right?

Reply:
I'm adding the column with this PR, so all the rows will have their `uuid` set to `null`. I don't see the need to make it more precise.

Reply:
Ah, sure. I thought this was being called somewhere else. Just now I read the class name.